<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmaaDialog</class>
 <widget class="QDialog" name="SmaaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>159</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SMAA effect settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QGroupBox" name="edgeDetection">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;smaaEdgeDetection changes the edge detection shader:&lt;/p&gt;&lt;p&gt;luma  - default&lt;/p&gt;&lt;p&gt;color - might catch more edges, but is more expensive&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>EdgeDetection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="edgeDetectionValue">
        <item>
         <property name="text">
          <string>luma</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>color</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="threshold">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;smaaThreshold specifies the threshold or sensitivity to edges&lt;/p&gt;&lt;p&gt;Lowering this value you will be able to detect more edges at the expense of performance.&lt;/p&gt;&lt;p&gt;Range: [0, 0.5]&lt;/p&gt;&lt;p&gt;0.1 is a reasonable value, and allows to catch most visible edges.&lt;/p&gt;&lt;p&gt;0.05 is a rather overkill value, that allows to catch 'em all.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>Threshold</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QDoubleSpinBox" name="thresholdValue">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="maxSearchSteps">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;smaaMaxSearchSteps specifies the maximum steps performed in the horizontal/vertical pattern searches&lt;/p&gt;&lt;p&gt;Range: [0, 112]&lt;/p&gt;&lt;p&gt;4  - low&lt;/p&gt;&lt;p&gt;8  - medium&lt;/p&gt;&lt;p&gt;16 - high&lt;/p&gt;&lt;p&gt;32 - ultra&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>MaxSearchSteps</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QSpinBox" name="maxSearchStepsValue">
        <property name="minimum">
         <number>-10000</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>32</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="maxSearchStepsDiag">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;smaaMaxSearchStepsDiag specifies the maximum steps performed in the diagonal pattern searches&lt;/p&gt;&lt;p&gt;Range: [0, 20]&lt;/p&gt;&lt;p&gt;0  - low, medium&lt;/p&gt;&lt;p&gt;8  - high&lt;/p&gt;&lt;p&gt;16 - ultra&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>MaxSearchStepsDiag</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QSpinBox" name="maxSearchStepsDiagValue">
        <property name="minimum">
         <number>-10000</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>16</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="cornerRounding">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;smaaCornerRounding specifies how much sharp corners will be rounded&lt;/p&gt;&lt;p&gt;Range: [0, 100]&lt;/p&gt;&lt;p&gt;25 is a reasonable value&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>CornerRounding</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QSpinBox" name="cornerRoundingValue">
        <property name="minimum">
         <number>-10000</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SmaaDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SmaaDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
